css/component_styles/badge.css

```diff
@@ -0,0 +1,54 @@
+/* ------- badge css -------- */
+
+.badge-container {
+    position: relative;
+}
+
+/* profile badge */
+.badge {
+    position: absolute;
+    width: 1rem;
+    height: 1rem;
+    border: solid 2px var(--bg-color);
+    border-radius: 50%;
+    background-color: var(--color-success);
+    right: 1rem;
+    bottom: 5px;
+}
+
+.offline {
+    background-color: var(--color-dnd);
+}
+
+.dnd {
+    background-color: var(--color-error);
+}
+
+/* number badge */
+.shopping-cart {
+    font-size: 2rem;
+}
+
+.badge-ico {
```

Comment:
badge-icon?

Reply:
Yaa ! 😅, actually there were two classes one badge-icon for the entire badge with icon and this badge-ico class for the badge on the icon